src/rgw/rgw_sal_motr.cc

```diff
@@ -387,7 +388,19 @@ int MotrUser::store_user(const DoutPrefixProvider* dpp,
   RGWObjVersionTracker objv_tr = {};
   obj_version& obj_ver = objv_tr.read_version;
 
-  ldpp_dout(dpp, 10) << "Store_user(): User = " << info.user_id.id << dendl;
+  std::string access_key;
+  std::string secret_key;
+  if (!info.access_keys.empty()) {
+    std::map<std::string, RGWAccessKey>::const_iterator iter = info.access_keys.begin();
+    const RGWAccessKey& k = iter->second;
+    access_key = k.id;
+    secret_key = k.key;
+    MGWAccessKey MGWUserKeys(access_key, secret_key, info.user_id.id);
+    store->store_access_key(dpp, y, MGWUserKeys);
+  }
+
+
+  ldpp_dout(dpp, 20) << "Store_user(): User = " << info.user_id.id << dendl;
```

Comment:
This line shd be at the same location, i.e. at 391

Reply:
done